doc/btp_spec.txt

```diff
@@ -3724,3 +3725,649 @@ Commands and responses:
+					UUID (2 or 16 octets)
+
+		Valid UUID_Length parameter values:
+					0x02 = UUID16
+					0x10 = UUID128
+
+		If Status is non-zero Characteristics_Count shall be set to 0.
+
+	Opcode 0x86 - Discover All Characteristic Descriptors Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+					Descriptors_Count (1 octet)
+					[array] Descriptor (variable)
+
+		Object Descriptor is defined as:
+					Descriptor_Handle (2 octets)
+					UUID_Length (1 octet)
+					UUID (2 or 16 octets)
+
+		Valid UUID_Length parameter values:
+					0x02 = UUID16
+					0x10 = UUID128
+
+		If Status is non-zero Descriptors_Count shall be set to 0.
+
+	Opcode 0x87 - Read Characteristic Value/Descriptor Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+					Data_Length (2 octets)
+					Data (variable)
+
+		If Status is non-zero Data_Length shall be set to 0.
+
+	Opcode 0x88 - Read Using Characteristic UUID Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+					Data_Length (2 octets)
+					Data (variable)
+
+		Valid UUID_Length parameter values:
+					0x02 = UUID16
+					0x10 = UUID128
+
+		If Status is non-zero Data_Length shall be set to 0.
+
+	Opcode 0x89 - Read Long Characteristic Value/Descriptor Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+					Data_Length (2 octets)
+					Data (variable)
+
+		If Status is non-zero Data_Length shall be set to 0.
+
+	Opcode 0x8a - Read Multiple Characteristic Values Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+					Data_Length (2 octets)
+					Data (variable)
+
+		If Status is non-zero Data_Length shall be set to 0.
+
+	Opcode 0x8b - Write Characteristic Value/Descriptor Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+
+	Opcode 0x8c - Write Long Characteristic Value/Descriptor Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+
+	Opcode 0x8d - Reliable Write Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+
+	Opcode 0x8e - Configure Notifications Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+
+	Opcode 0x8f - Configure Indications Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+
+	Opcode 0x90 - Notification/Indication Received
```

Comment:
support for multiple handle values notification - either new even or extend this with new type (and possible explicit count)